session/bootstrap_test.go

```diff
@@ -318,3 +318,21 @@ func (s *testBootstrapSuite) TestBootstrapInitExpensiveQueryHandle(c *C) {
 	dom.InitExpensiveQueryHandle()
 	c.Assert(dom.ExpensiveQueryHandle(), NotNil)
 }
+
+func (s *testBootstrapSuite) TestStmtSummary(c *C) {
+	defer testleak.AfterTest(c)()
+	ctx := context.Background()
+	store, dom := newStoreWithBootstrap(c, s.dbName)
+	defer store.Close()
+	defer dom.Close()
+	se := newSession(c, store, s.dbName)
+	mustExecSQL(c, se, `update mysql.global_variables set variable_value='' where variable_name='tidb_enable_stmt_summary'`)
```

Comment:
you can use fail point to skip the code in Bootstrap,  if you want to test the correctness of `convertStmtSummaryVars` function manually.

Reply:
It's not possible to ensure that the origin global variables are empty before boostrap. It should work only when they're empty.

Reply:
ok